packages/flutter/lib/src/rendering/layout_helper.dart

```diff
@@ -11,7 +11,8 @@ import 'box.dart';
 /// [BoxConstraints].
 ///
 /// The methods of [ChildLayoutHelper] adhere to this signature.
-typedef ChildLayouter = Size Function(RenderBox child, BoxConstraints constraints);
+typedef ChildLayouter = Size Function(
+    RenderBox child, BoxConstraints constraints);
```

Comment:
Aim for a maximum line length of roughly 80 characters, but prefer going over if breaking the line would make it less readable.
Refer to [Style Guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters)

Reply:
very sorry to make this style error. changing this by making another commit.